src/QtAVPlayer/qavplayer.cpp

```diff
@@ -274,9 +274,17 @@ void QAVPlayerPrivate::doDemux()
         }
 
         if (packet.streamIndex() == demuxer.videoStream())
+        {
             videoQueue.enqueue(packet);
+            if(state == QAVPlayer::PausedState && isVideoAvailable()) {
```

Comment:
isVideoAvailable() is not needed, since there is a check against demuxer.videoStream()

Reply:
ah, right :)